src/clj/rems/db/form.clj

```diff
@@ -83,13 +83,23 @@
                                       {:field/id "xyz"}
                                       {:field/id "abc"}])))))
 
+(defn- normalize-field-values [field]
```

Comment:
the name sounds like it's talking about `:field/values`. how about something like `normalize-field-definitions`? or perhaps just `normalize`?